cli/azd/internal/telemetry/fields/fields.go

```diff
@@ -85,6 +85,7 @@ const (
 	EnvGoogleCloudBuild   = "Google Cloud Build"
 	EnvTeamCity           = "TeamCity"
 	EnvJetBrainsSpace     = "JetBrains Space"
+	Codespaces            = "GitHub Codespaces"
```

Comment:
nit: the environments are prefixed with `Env`, so this should be `EnvCodespaces`

Reply:
🤦